bats/foreign-keys.bats

```diff
@@ -962,7 +977,7 @@ SET FOREIGN_KEY_CHECKS=0;
 UPDATE parent SET v1 = v1 - 1;
 SQL
     dolt add -A
-    dolt commit -m "updated parent"
+    dolt commit --force -m "updated parent"

```

Comment:
Why do we need these --forces now?

Reply:
Previously we didn't check row data on commit (only schema information), but now we do. These tests are verifying that the merge behavior is working correctly when working with invalid table state, so now to commit bad tables we have to force them.

Reply:
Understood

Reply:
I see. All these have `SET FOREIGN_KEY_CHECKS=0;` above.